crates/video/Cargo.toml

```diff
@@ -0,0 +1,41 @@
+[package]
+name = "vhost-user-video"
+version = "0.1.0"
+authors = ["Albert Esteve <aesteve@redhat.com>"]
+description = "A virtio-video device using the vhost-user protocol."
+repository = "https://github.com/rust-vmm/vhost-device"
+readme = "README.md"
+keywords = ["vhost", "video", "virt", "backend"]
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[features]
+default = ["v4l2-decoder"]
+v4l2-decoder = ["v4l2r", "nix"]
+
+[dependencies]
+bitflags = "2.3.3"
+clap = { version = "4.3",  features = ["derive"] }
+env_logger = "0.10"
+epoll = "4.3"
+enumn = "0.1.6"
```

Comment:
SCSI uses `num_enum`. I have no preference over one or the other. But we should probably decide on one crate for the task.

Reply:
I agree, and I think is an easy decision: use the one that was already used in the project (i.e., `num_enum`).

I am relatively new to Rust and picked `enumn` because I found it first. But if there is a similar crate already being used, I see not big reason why I should open a debate to change it now :)